domain/src/test/java/org/oppia/domain/UserAppHistoryControllerTest.kt

```diff
@@ -58,7 +58,8 @@ class UserAppHistoryControllerTest {
   @JvmField
```

Comment:
There are no changes in this file, please revert this file.

Reply:
Done